src/python/dxpy/scripts/dx.py

```diff
@@ -4881,6 +4884,7 @@ def register_parser(parser, subparsers_action=None, categories=('other', ), add_
                         help=fill('Print help and examples for how to specify inputs',
                                   width_adjustment=-24),
                         action=runInputHelp, nargs=0)
+parser_run.add_argument('--detach', help=fill("New job will not start as subjob if run inside of a different job.", width_adjustment=-24), action='store_true')
```

Comment:
I think we can use a 'detached job' expression, so for example: "New job will start as a detached job if run inside of a different job", to tell the user what the new job *will* be launched as.

Reply:
I did not want to use the word "detached" to describe what "detach" means.  But sure, I'll update it